### ISSUE TITLE
Bypass Google Forms integration by using a flag

### ISSUE THREAD

**Issue by sporrera** [GatorGrouper/gatorgrouper#74]
To bypass the Google Forms integration and instead supply a list of students
directly to the program, use the `--students-file` flag.

```shell
python3 gatorgrouper.py --students-file "students_list.csv"
python3 gatorgrouper.py --students-file students_list.csv
```

These commands do not work.  I get the following error when I run both commands: 
"File "gatorgrouper.py", line 42, in <module>
    SHUFFLED_STUDENT_IDENTIFIERS = shuffle_students(STUDENT_IDENTIFIERS) line 47, in shuffle_students shuffle(shuffled_responses File"/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/random.py", line 274, in shuffle x[i], x[j] = x[j], x[i]
TypeError: 'str' object does not support item assignment"

**Comment by dillam** [comment]
@sporrera Whenever you run those commands are you copy and pasting from the readme?

**Comment by sporrera** [comment]
I have both copied and pasted the commands and typed them out individually

**Comment by dillam** [comment]
When you use them, is there a file in the directory called `students_list.csv`?

**Comment by ZachAndrews98** [comment]
I ran the bottom command (python3 gatorgrouper.py --students-file students_list.csv) with a file named students_list.csv and I did not get any errors. This was my output (the first line shows the name of the file being read): 
students_list.csv
Group 1
sutterj
eltmanm
thomast2
vargask

Group 2
colez
toccin
millinj2

Group 3
burrowss
bonnettd
shafferz

Group 4
lylet
cerdamejiaj
kimy

Group 5
wojnarn
luoman
bristola

Group 6
delgrecoj
andrewsz
bryana

Group 7
kimy
dillam
mendezj

**Comment by sporrera** [comment]
No, and after a recent pull I only have students.csv .. If I change the command to 

```
python3 gatorgrouper.py --students-file "students.csv
```

then I can run the command

**Comment by cerdamejiaj** [comment]
The readme file is not clear enough. Instead of having `python3 gatorgrouper.py --students-file "students_list.csv"`  

we should have 
`python3 gatorgrouper.py --students-file "filename.csv"` 

something that is better to understand.

**Comment by dillam** [comment]
Okay, create a pull request then I will close the issue.

**Comment by sporrera** [comment]
addressed in pull request #82
